manifest.webapp

```diff
@@ -64,6 +64,16 @@
       "type": "io.cozy.settings",
       "verbs": ["GET", "PUT"]
     },
+    "profiles": {
+      "description": "Required to manage profiles on Pass extension",
+      "type": "com.bitwarden.profiles",
```

Comment:
The web app is already able to display/edit profiles. Am i missing something on how permissions work here?